frontend/src/metabase/dashboard/components/add-card-sidebar/QuestionPickerItem.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { ItemRoot, ItemIcon, ItemTitle } from "./QuestionPickerItem.styled";
+
+QuestionPickerItem.propTypes = {
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  icon: PropTypes.string.isRequired,
+  isCollection: PropTypes.bool,
+  onSelect: PropTypes.func.isRequired,
+};
+
+export function QuestionPickerItem({ id, name, icon, isCollection, onSelect }) {
+  return (
+    <ItemRoot onClick={() => onSelect(id)}>
```

Comment:
I think we should stick a button in this `li` and put the `onClick` listener on it so that this menu is more accessible. Another alternative, I think, is the `menuitem` role + setting `tabIndex={0}`: https://w3c.github.io/aria/#menuitem